app/src/main/java/com/fidesmo/ble/client/apdu/CustomPacketDefragmenter.java

```diff
@@ -0,0 +1,16 @@
+package com.fidesmo.ble.client.apdu;
+
+
+public interface CustomPacketDefragmenter {
```

Comment:
Maybe we can think on how to merge this `CustomPacketDefragmenter` with original Packet fragmenter/defragmenter interfaces? Some of those methods can be handy there too.

But I have some notes about the interface: 
* Method `add` is not informative, looking at the interface you can't say what it is doing and how it is different from the `append`. Maybe we can rename both to something like `addPacket` and `addBuffer` (or nay other variant)?
* Interface is not consistent: you have Java Bean style `getBuffer` and simplified `complete` (which in turn has to be `completed` maybe?) and `empty`. IHMO it would be good to choose one naming scheme or append `isCompleted` or remove `get` what is better to you.
* Github complains about not having newline at the end